kornia/color/lab.py

```diff
@@ -0,0 +1,217 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+from typing import Tuple
+
+"""
+# Adapted from https://github.com/xahidbuffon/rgb-lab-conv/blob/master/rgb_lab_formulation.py and based on:
+https://github.com/affinelayer/pix2pix-tensorflow/blob/master/pix2pix.py
+https://github.com/torch/image/blob/9f65c30167b2048ecbe8b7befdc6b2d6d12baee9/generic/image.c
+https://github.com/cameronfabbri/Colorizing-Images-Using-Adversarial-Networks
+"""
+
+import sys 
+sprintf = lambda format, *values : format % values
+printf  = lambda format, *values : sys.stdout.write(format % values)
+
+class RgbToLab(nn.Module):
+    r"""Convert image from RGB to LAB
+    The image data is assumed to be in the range of (0, 1).
+    args:
+        image (torch.Tensor): RGB image to be converted to LAB.
+    returns:
+        torch.tensor: LAB version of the image.
+    shape:
+        - image: :math:`(*, 3, H, W)`
+        - output: :math:`(*, 3, H, W)`
+    Examples::
+        >>> input = torch.rand(2, 3, 4, 5)
+        >>> lab = kornia.color.RgbToLab()
+        >>> output = lab(input)  # 2x3x4x5
+    Reference::
+        [1] 
+    """
+
+    def __init__(self) -> None:
+        super(RgbToLab, self).__init__()
+
+    def forward(  # type: ignore
+            self, input: torch.Tensor) -> torch.Tensor:
+        return rgb_to_lab(input)
+
+    
+def srgb_to_xyz(input: torch.Tensor) -> torch.Tensor:
+    srgb_pixels = input
+    
+    # Constant tensor used for conversion
```

Comment:
Reference for conversion  matrix?

Reply:
agree. lets block this one until tests are provided